src/sentry/utils/performance_issues/performance_detection.py

```diff
@@ -155,6 +162,8 @@ def _detect_performance_problems(data: Event, sdk_span: Any) -> List[Performance
     for span in spans:
```

Comment:
you might want to update `used_perf_issue_detectors` to include only `DetectorType.N_PLUS_ONE_DB_QUERIES` because that's what's used to generate detector's output

Reply:
:+1: Done!